gateway/src/apicast/configuration/service.lua

```diff
@@ -30,13 +33,42 @@ local http_methods_with_body = {
   PATCH = true
 }
 
+local function is_http2_request()
+  return ngx.req.http_version == '2.0'
+end
+
+local function content_type_is_urlencoded(headers)
+    local ct = headers["Content-Type"]
```

Comment:
As I remember when running on Openshift, headers are all lower case. so perhaps use `ngx.var.http_content_type` instead?

Reply:
From https://github.com/openresty/lua-nginx-module/tree/v0.10.19#ngxreqhttp_version

```
Also, by default, an __index metamethod is added to the resulting Lua table and will normalize the keys to a pure lowercase form with all underscores converted to dashes in case of a lookup miss. For example, if a request header My-Foo-Header is present, then the following invocations will all pick up the value of this header correctly:
 ngx.say(headers.my_foo_header)
 ngx.say(headers["My-Foo-Header"])
 ngx.say(headers["my-foo-header"])
```